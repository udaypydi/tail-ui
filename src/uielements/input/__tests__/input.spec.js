import React from 'react';
import { render } from 'test-utils';
import  Input from 'uielements/input/input.component.js';

it('should take a snapshot', () => {
    const { asFragment } = render(<Input />);
    
    expect(asFragment(<Input />)).toMatchSnapshot();
});
