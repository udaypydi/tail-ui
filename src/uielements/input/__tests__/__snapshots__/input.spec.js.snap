// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <input
    class="bg-white focus:outline-none focus:shadow-outline border border-gray-300 py-2 px-4 block w-full appearance-none leading-normal"
    placeholder=""
    type="string"
  />
</DocumentFragment>
`;
